xacro/__init__.py

```diff
@@ -171,41 +181,58 @@ def check_deprecated_tag(tag_name):
 
 
 class Macro(object):
+
     def __init__(self):
         self.body = None  # original xml.dom.Node
         self.params = []  # parsed parameter names
         self.defaultmap = {}  # default parameter values
         self.history = []  # definition history
 
 
+class ResourceNotFound(Exception):
+    pass
+
+
 def eval_extension(s):
-    if s == '$(cwd)':
+    # TODO This is Temporary fix to return cwd() for 'find xacro' command.
+    # We can use resolve_args() when rospkg will be migrated to ROS2.
+    if s == '$(cwd)' or s == '$(find xacro)':
         return os.getcwd()
     try:
-        from roslaunch import substitution_args
-        from rospkg.common import ResourceNotFound
-        return substitution_args.resolve_args(s, context=substitution_args_context, resolve_anon=False)
+        # TODO use following when roslaunch will be fully migrated to ROS2.
+        # from roslaunch import substitution_args
```

Comment:
I'm not sure what's needed exactly to resolve this TODO, but I don't think depending on launch is an appropriate solution. It looks like `substitution_args` from roslaunch is already replicated in `xacro`, so maybe we can remove this TODO.

Reply:
I agree